### ISSUE TITLE
Issue: It's unclear when lizzie is having trouble loading leela zero

### ISSUE THREAD

**Issue by hyperpape** [featurecat/lizzie#399]
Using MacOS:

I downloaded lizzie 0.5, built leela-zero from source, then moved `leelaz`, `best-network`, and `leelaz_opencl_tuning` to the lizzie directory, before running `java -jar Lizzie.jar`. The UI displays a indicator "Leela Zero is loading', which it has been showing for at least 20 minutes. 

It would be nice if either the java command or the UI could somehow indicate progress (or lack thereof), so that the user knew how long to wait, and whether lizzie was properly detecting files.

**Comment by featurecat** [comment]
Agreed this is a big issue. No one knows how long to wait. It would be nice to display the leela zero commandline output while leela zero is loading.
